__tests__/app.test.js

```diff
@@ -379,8 +379,75 @@ describe("app", () => {
         });
         });
     });
+  });
+
+  describe("10. GET /api/articles (queries)", () => {
+    test("it should return an array of articles objects filtered by a givin topic sorted in ascending order by article_id", () => {
+      const topic = "mitch";
+      return request(app)
+        .get(`/api/articles?topic=mitch&order=ASC&sortBy=article_id`)
+        .expect(200)
+        .then((data) => {
+          expect(data.body.length).toBe(11);
+          expect(data.body[0].article_id).toBe(1);
+          expect(data.body[data.body.length-1].article_id).toBe(12);
+          data.body.forEach((topic) => {
+          expect(topic.topic).toBe("mitch");
+        
+          });
+        });
+    });
   
-    
+    test("it should return an array of articles objects filtered by a givin topic sorted and ordered by the defult values", () => {
+      const topic = "mitch";
+      return request(app)
+        .get(`/api/articles?topic=mitch`)
+        .expect(200)
+        
+        .then((data) => {
+         
+          expect(data.body.length).toBe(11);
+          expect(data.body[0].created_at).toBe('2020-11-03T09:12:00.000Z');
+          expect(data.body[data.body.length-1].created_at).toBe('2020-01-07T14:08:00.000Z');
+          data.body.forEach((topic) => {
+          expect(topic.topic).toBe("mitch");
+          
+          });
+
+        })
+       
```

Comment:
There should also be seperate tests for when you just query order, and when you just query for sort_by as well. This is to ensure these queries work in isolation as well